testproject/Assets/Tests/Manual/Scripts/RandomMovement.cs

```diff
@@ -17,29 +17,87 @@ public override void OnNetworkSpawn()
             m_Rigidbody = GetComponent<Rigidbody>();
             if (NetworkObject != null && m_Rigidbody != null)
             {
-                m_Rigidbody.isKinematic = !NetworkObject.IsOwner;
-                if (!m_Rigidbody.isKinematic)
+                if (NetworkObject.IsOwner)
```

Comment:
`IsServer` here too?

Reply:
No, that is just to initialize the direction randomly.